TravelerApp/TravelerAppCore/Controller/SearchByLocalisation.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using TravelerAppCore.Models.Hotels;
+using TravelerAppCore.View;
+
+namespace TravelerAppCore.Models
+{
+    public static class SearchByLocalisation
+    {
+        public static List<Root> byLocalisation(List<Root> targetData, string address)
+        {
+            List<Root> hotelLocalisation = new List<Root>();
+            string regPattern = $@"{address}";
+            Regex regEx = new Regex(regPattern, RegexOptions.IgnoreCase);
+
+            foreach (Root hotel in targetData)
+            {
+                if (hotel.HotelInfo.Address == null)
+                {
+                    continue;
+                }
+
+                if (regEx.IsMatch(hotel.HotelInfo.Address))
+                {
+                    hotelLocalisation.Add(hotel);
+                }
+            }
+            return hotelLocalisation;
+        }
+    }
+
+    public class consoleAddress
```

Comment:
Nazwa klasy PascalCase. Dodatkowo warto się zastanowić czy projekt Core jest dobrym miejscem to trzymanie tego typu funkcjonalności. Core to ma być projekt w którym są podstawowe funkcjonalności nie zależne od typu aplikacji jaki tworzymy a w tym przypadku używasz funkcjonalności przeznaczonych dla aplikacji konsolowej :)

Reply:
Nazwy poprawione, przeniosłem wszystko co odpowiadało rysowaniu i wypisywaniu na console na razie do katalogu View gdzie znajduje się też metoda rysowania tabeli, czy nie przenieść tego dokładnie do TravelerAppConsole to trzeba by było ustalić już z całym zespołem